_docs/services/aws-elasticsearchBETA.md

```diff
@@ -89,6 +89,109 @@ For customers that would like to import or export their Elasticsearch data, this
 
  cloud.gov does offer a code sample repository on Github - [aws-elasticsearch-example](https://github.com/cloud-gov/aws-elasticsearch-example) that shows an example in Python on how to interact with the new ES service using signed headers.  Our customers are encouraged to submit PRs of other examples to share with fellow customers.
 
+## Migration from Kubernetes Elasticsearch 5.6 to AWS Elasticsearch 7.4
+This content is only applicable to customers using the pre-AWS Elasticsearch service on cloud.gov during the transition period before that service is fully deprecated in 2021.
+
+We recommend that you build new document indices in AWS Elasticsearch from your source documents instead of migrating the data and indices from the existing service to the new service. The migration is experimental could result in more time and effort than a rebuild would entail.
+
+The overall process looks like this:
+
+1. Ask Cloud.gov Support to access the AWS Elasticsearch plan named `BETA-es-dev-6.8-migration`
+2. Take a snapshot of your ES 5.6 index and store in an s3 bucket.
+3. Create ES 6.8 service instance and create a service key with `s3-bucket` info attached (See Above Section)
+4. Restore ES 5.6 Snapshot into your ES 6.8
+5. Upgrade your ES 6.8 to 7.4
+
+`Note: Steps 2-4 is done by the user`
+
+### Prepare for Migration
+* Have a S3 service instance created with service key - [S3 Service Instructions](https://cloud.gov/docs/services/s3/)
+* Have an existing ES 5.6 instance with snapshots to S3 - [ES 5.6 Backups Instructions](https://cloud.gov/docs/services/elasticsearch56/#managing-backups)
+* Ask Cloud.gov Support to have access to migration plan `BETA-es-dev-6.8-migration`
+* Create new AWS Elasticsearch service instance with `BETA-es-dev-6.8-migration` plan
+* Create Service Key with attached S3 - [ES 7.4 Service Key](https://cloud.gov/docs/services/aws-elasticsearchBETA/#managing-backups)
+
+### Backup your ES 5.6 to S3 Bucket
+* Connect to your Elasticsearch service using port forwarding. **Note**: You'll need to leave the [`cf ssh`]({{ site.baseurl }}{% link _docs/management/using-ssh.md %}) command running and follow the next steps in a different terminal so that you can access the remote Elasticsearch instance from your local environment:
+
+    ```sh
+    es_credentials=$(cf service-key my-elastic-service my-key | tail -n +3)
+
+    es_hostname=$(echo "${es_credentials}" | jq -r '.hostname')
+    es_port=$(echo "${es_credentials}" | jq -r '.port')
+    es_username=$(echo "${es_credentials}" | jq -r '.username')
+    es_password=$(echo "${es_credentials}" | jq -r '.password')
+
+    cf ssh my-app -L "9200:${es_hostname}:${es_port}"
+    ```
+
```

Comment:
Adding note back here:

```suggestion
**Note**: You'll need to leave this [`cf ssh`]({{ site.baseurl }}{% link _docs/management/using-ssh.md %}) command running and perform the following steps a different terminal so that you can access the remote Elasticsearch instance from your local environment.

```